lib/crate_ruby/client.rb

```diff
@@ -83,13 +83,13 @@ def drop_table(table_name, blob=false)
     # List all user tables
     # @return [ResultSet]
     def show_tables
-      execute("select table_name from information_schema.tables where schema_name = 'doc'")
+      execute("select table_name from information_schema.tables where table_schema = 'doc'")
```

Comment:
This breaks compatibility with older Crate versions - would it make sense to add a compatibility matrix in the README?

Reply:
Probably best someone of you guys do it, as I have no clue when your change was introduced. ...